### PR TITLE
Custom countries #4086

### PR DESCRIPTION
**Pull request by roryc89**
addresses main app issue #4086

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,105 @@
+0 info it worked if it ends with ok
```
Comment (Conorc1000) [inline npm-debug.log]: Do we want to commit this log?

File: src/Main.elm [hunk 2]
```diff
@@ -22,9 +22,8 @@ init flags =
         authors =
             Decoders.authors flags.authorsList
 
-        { affiliationLimit, authorLimit, showInstitution, showCity, showState, showCountry, mandatoryInstitution,
-          mandatoryCity, mandatoryState, mandatoryCountry, class, institutionLabel, cityLabel, stateLabel, countryLabel}
```
Comment (Conorc1000) [inline src/Main.elm]: Why are we getting rid of the affiliation labels?

File: src/Main.elm [hunk 3]
```diff
@@ -61,11 +60,8 @@ init flags =
                 , mandatoryCity = mandatoryCity
                 , mandatoryState = mandatoryState
                 , mandatoryCountry = mandatoryCountry
-                , institutionLabel = institutionLabel
-                , cityLabel = cityLabel
-                , stateLabel = stateLabel
-                , countryLabel = countryLabel
```
Comment (Conorc1000) [inline src/Main.elm]: Dito

**Comment by Conorc1000** [comment]
This is what i see when i run the app locally 
screen shot 2017-09-27 at 10 52 16

File: npm-debug.log (hunk 1 above)
Comment (roryc89) [reply npm-debug.log]: Good spot

File: src/Main.elm (hunk 2 above)
Comment (roryc89) [reply src/Main.elm]: looks like funny merge business

**Pull request merged**